src/api_server/swagger/firecracker.yaml

```diff
@@ -399,6 +399,81 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /snapshot/create:
+    put:
+      summary: Creates a full or diff snapshot. Post-boot only.
```

Comment:
microVM needs to be in the Paused state.

Reply:
Done, I added this in description to keep the summary short.